cmd/podman/prune.go

```diff
@@ -0,0 +1,100 @@
+package main
```

Comment:
Can you split this into two different go files
containers_prune.go
images_prune.go